audit/projects/kubernetes-public/services/container/clusters/aaa.json

```diff
@@ -163,6 +179,10 @@
         "maxPodsPerNode": "110"
       },
       "name": "pool1-20190918234853194900000001",
+      "networkConfig": {
+        "podIpv4CidrBlock": "10.40.0.0/14",
+        "podRange": "gke-aaa-pods-4d0eb763"
+      },
```

Comment:
@ameukam I never saw anything about this when running `terraform` so I'm going to assume this was GKE auto-applying this change